src/importer/util_azure.py

```diff
@@ -8,22 +8,58 @@
 
 log = logging.getLogger(__name__)
 
-# TODO: Make version to works locally
-def get_container_client(container_name):
-    default_credential = DefaultAzureCredential()
-    blob_service_client = BlobServiceClient(settings.STORAGE_ACCOUNT_URL, credential=default_credential)
+def get_container_client():
+    if settings.AZURITE_STORAGE_CONNECTION_STRING:
+        blob_service_client = BlobServiceClient.from_connection_string(settings.AZURITE_STORAGE_CONNECTION_STRING)
+    else:
+        default_credential = DefaultAzureCredential()
+        blob_service_client = BlobServiceClient(settings.STORAGE_ACCOUNT_URL, credential=default_credential)
+
+    return blob_service_client
+
+def get_blob_container_client(container_name):
+    blob_service_client = get_container_client()
     container_client = blob_service_client.get_container_client(container=container_name)
     return container_client
 
-def download_xml_files():
-    if os.path.exists(settings.DATA_DIR):
-        shutil.rmtree(settings.DATA_DIR)
-    os.makedirs(settings.DATA_DIR)
+def get_blob_client(container_name, blob_name):
+    blob_service_client = get_container_client()
+    blob_client = blob_service_client.get_blob_client(container=container_name, blob=blob_name)
+    return blob_client
+
+def download_blob_to_file(container_name, blob_name, output_dir):
```

Comment:
Je zou hier nog een arg aan toe kunnen voegen die defined of die output_dir gewiped moet worden. In de vorige setup was dat altijd nodig, maar de naam van deze function suggereert niet per sé dat die output dir gewiped wordt.

Reply:
Hij wordt niet meer gewiped. Dat doe ik nu in de calling functie. Hij maakt nu alleen de folder aan als deze nog niet bestaat